src/components/card/Card.tsx

```diff
@@ -21,27 +20,17 @@ const Card: React.FC<ICardProps> = ({
   const { loggedIn } = useContext<IContext>(UserContext);
   const [flowerInfo, setFlowerInfo] = useState<IExampleFlower>(flowerData);
 
+  const dispatch = useDispatch();
+  const { deleteOneFavFlower } = bindActionCreators(actionCreators, dispatch);
```

Comment:
you can import deleteOneFavFlower, 
and try dispatach(deleteOneFavFlower()) don't need bindActionCreators